### PR TITLE
feat(rename): rename to StrykerJS

### LINKED ISSUE #2754: Rename "Stryker" to "StrykerJS"
It's time...

When we started with Stryker back in 2014, there was no Stryker.NET or Stryker4s (hack, there was no webpack for that matter). Since then, we're maintaining 3 mutation testing frameworks, all under the "Stryker Mutator" flag 👽🏳. 

However, it is more and more common to refer to all three as "Stryker" (instead of "Stryker Mutator"). It is very common to refer to Stryker itself as "StrykerJS". I myself have caught myself doing exactly that.

I think it is time to rebrand Stryker as StrykerJS. 

@stryker-mutator/stryker-js-and-friends-maintainers what do you think?

The website URL should also be updated: 

```diff
-https://stryker-mutator.io/docs/stryker
+https://stryker-mutator.io/docs/stryker-js
```

- [x] Update `custom_edit_url`'s in docs
- [x] Update links in the readme's
- [x] Update headers in the readmes (Stryker -> StrykerJS)
- [x] Update links in the package.json
- [x] Update guide links in the custom initializes (react, vue, angular)
- [ ] Update website
    - [ ] Configure redirect link in docusaurus
- [ ] Rename repository (stryker -> stryker-js)

### PR DESCRIPTION
**Pull request by nicojs**
Rename the "stryker" github repository to "stryker-js"

* Stryker -> StrykerJS in readmes and docs.
* https://stryker-mutator/docs/stryker -> https://stryker-mutator/docs/stryker-js
* https://github.com/stryker-mutator/stryker -> https://github.com/stryker-mutator/stryker-js
* https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker/master -> https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker-js/master
* Add `repository.directory` to package.json's where it was missing.

Note: the packages themselves are not renamed, they're still called `stryker-mutator/core`, etc.

Closes #2754

### CODE REVIEW

File: packages/jasmine-runner/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+[![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fstryker-mutator%2Fstryker%2Fmaster%3Fmodule%3Djasmine-runner)](https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker-js/master?module=jasmine-runner)
+[![Build Status](https://github.com/stryker-mutator/stryker-js/workflows/CI/badge.svg)](https://github.com/stryker-mutator/stryker-js/actions?query=workflow%3ACI+branch%3Amaster)
 [![NPM](https://img.shields.io/npm/dm/@stryker-mutator/jasmine-runner.svg)](https://www.npmjs.com/package/@stryker-mutator/jasmine-runner)
 [![Node version](https://img.shields.io/node/v/@stryker-mutator/jasmine-runner.svg)](https://img.shields.io/node/v/@stryker-mutator/jasmine-runner.svg)
 [![Slack Chat](https://img.shields.io/badge/slack-chat-brightgreen.svg?logo=slack)](https://join.slack.com/t/stryker-mutator/shared_invite/enQtOTUyMTYyNTg1NDQ0LTU4ODNmZDlmN2I3MmEyMTVhYjZlYmJkOThlNTY3NTM1M2QxYmM5YTM3ODQxYmJjY2YyYzllM2RkMmM1NjNjZjM)
 
-# Stryker Jasmine Runner
+# StrykerJS Jasmine Runner
```
Comment (simondel) [inline packages/jasmine-runner/README.md:7]: Perhaps just 'Jasmine Runner'?

File: packages/jasmine-runner/package.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com/stryker-mutator/stryker.git"
+    "url": "git+ssh://git@github.com/stryker-mutator/stryker.git",
```
Comment (simondel) [inline packages/jasmine-runner/package.json:22]: Shouldn't this be `stryker-js.git`?

**Review by simondel — CHANGES_REQUESTED** [review]

File: packages/jasmine-runner/README.md (hunk 1 above)
Comment (simondel) [reply packages/jasmine-runner/README.md:7]: (Same goes for the other packages)

File: packages/jasmine-runner/package.json (hunk 2 above)
Comment (simondel) [reply packages/jasmine-runner/package.json:22]: (Same goes for the other packages)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
-# Contribute to Stryker
+# Contribute to JS
```
[the remark this answers is not in the document]
Comment (nicojs) [reply CONTRIBUTING.md:1]: Sure, lower the scope of our project. 🙄

File: packages/jasmine-runner/README.md (hunk 1 above)
Comment (nicojs) [reply packages/jasmine-runner/README.md:7]: It isn't the jasmine test runner itself, right?

File: packages/jasmine-runner/package.json (hunk 2 above)
Comment (nicojs) [reply packages/jasmine-runner/package.json:22]: yes

File: packages/jasmine-runner/README.md (hunk 1 above)
Comment (simondel) [reply packages/jasmine-runner/README.md:7]: hmm fair enough

**Review by simondel — APPROVED** [review]

**Pull request merged**